specs/CalendarView/CalendarViewSpec1.md

```diff
@@ -0,0 +1,716 @@
+control's rendering in upcoming Windows OS releases.
+These properties will be used in WinUI to align the CalendarView's styling with the rest of the
+standard controls.
+
+With this change, `CalendarView` still has its existing visualization (styles/templates) that
+don't use these properties, and new ones that do.
+By default, the older visualization is used.
+The new visualization can be enabled by setting a boolean resource value
+(see `CalendarViewBaseItemRoundedChromeEnabled` below).
+And the `WinUI2`
+[controls library](https://www.nuget.org/packages/Microsoft.UI.Xaml/))
+sets this boolean. The end result is that these properties are ignored by default, but
+come into play when an app is using WinUI2.
+
+# Conceptual
+
+## Today styling
+
+The following properties of type
+[Brush](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Media.Brush)
+are used to customize the current day's rendering when `CalendarView.IsTodayHighlighted` is True:
+`TodayBackground`, `TodayBlackoutBackground`, `TodayBlackoutForeground`,
+`TodayHoverBackground`, `TodayPressedBackground`, `TodaySelectedInnerBorderBrush`.
+
+### Showcasing the TodayBackground property in the month, year and decade views
+
+The value of the `TodayBackground` property is used in the month, year and decade display modes
+of the `CalendarView`:
+
+```xml
+<CalendarView TodayBackground='{StaticResource AccentFillColorDefaultBrush}'/>
+```
+
+![Default Today Month Background.](images/TodayMonthBackground.png)
+![Default Today Year Background.](images/TodayYearBackground.png)
+![Default Today Decade Background.](images/TodayDecadeBackground.png)
+
+### Showcasing the TodayHoverBackground property - current date is hovered
+
+The value of the `TodayHoverBackground` property is used when the user
+hovers a pointer over the current date:
+
+```xml
+<CalendarView TodayHoverBackground='{StaticResource AccentFillColorSecondaryBrush}'/>
+```
+
+![Hover Today Background.](images/HoverTodayBackground.png)
+
+### Showcasing the TodayPressedBackground property
+
+The value of the `TodayPressedBackground` property is used when the user
+clicks/taps the current date:
+
+```xml
+<CalendarView TodayPressedBackground='{StaticResource AccentFillColorTertiaryBrush}'/>
+```
+
+![Pressed Today Background.](images/PressedTodayBackground.png)
+
+### Showcasing the `TodaySelectedInnerBorderBrush` property - current date is selected
+
+The example sets the color to be used on the inner border for the current date:
+
+```xml
+<CalendarView TodaySelectedInnerBorderBrush='{StaticResource TextOnAccentFillColorPrimaryBrush}'/>
+```
+
+![Selected Today Border.](images/SelectedTodayBorder.png)
+
+## Blacked Out styling
+
+The following properties of type
+[Brush](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Media.Brush)
+are used to customize the calendar items that are blacked out in the month view:
+`BlackoutForeground`, `TodayBlackoutBackground`, `TodayBlackoutForeground`, `BlackoutStrikethroughBrush`.
```

Comment:
Okay, I reverse-engineered the naming pattern: Brushes end in "Brush" except for Foreground and Background. Did I get that right?

Reply:
Yes

Reply:
Discussion: We've got some inconsistency unfortunately, but this aligns as well as possible with precedent.

Recommend: See if we can describe a pattern to follow going forward.